llama-index-integrations/readers/llama-index-readers-jira/llama_index/readers/jira/base.py

```diff
@@ -91,14 +96,26 @@ def load_data(
         epic_descripton = ""
 
         for issue in relevant_issues:
-            # Iterates through only issues and not epics
```

Comment:
This didn't actually iterate only through issues and not epics. It also iterated through epics, and added wrong metadata (the one of the previous non-epic issue)